internal/controllers/app_controller.go

```diff
@@ -271,6 +271,7 @@ func (cli workloadClient) Get(ctx context.Context) (*workload, error) {
 		if err != nil {
 			return nil, err
 		}
+		fmt.Println("...workload status", o.Status)
```

Comment:
did ya forget to remove this print statement?

Reply:
You don't want to read all my debugs?